pandas/core/strings.py

```diff
@@ -680,12 +680,33 @@ def str_repeat(arr, repeats):
 
     Parameters
     ----------
-    repeats : int or array
+    repeats : int or array of int
         Same value for all (int) or different value per (array)
 
     Returns
     -------
-    repeated : Series/Index of objects
+    Series  or Index of objects
+
```

Comment:
Two spaces after Series. Use `object` as the Python type instead of objects (like using `list of int` instead of `list of ints`.

Can you add a description in the line after the type, on what is being returned please?